cub/cub/device/device_reduce.cuh

```diff
@@ -434,9 +433,7 @@ struct DeviceReduce
       d_out,
       static_cast<OffsetT>(num_items),
       ::cuda::minimum<>{},
-      // replace with
-      // std::numeric_limits<T>::max() when
-      // C++11 support is more prevalent
+      // TODO(bgruber): replace with ::cuda::std::numeric_limits<T>::max() (breaking change)
```

Comment:
I thought we were breaking things?

Reply:
I wanted to separate non-breaking from breaking changes, so we can more easily roll if needed, and have a smaller diff to inspect with more attention.